tinygrad/tensor.py

```diff
@@ -529,31 +527,27 @@ def mish(self): return self * self.softplus().tanh()
   def softplus(self, beta=1): return (1/beta) * (1 + (self*beta).exp()).log()
   def softsign(self): return self / (1 + self.abs())
 
-  # ***** broadcasted binary mlops *****
+  # ***** broadcasted non-unary mlops *****
 
-  def _broadcasted(self, fxn:Type[Function], other:Union[Tensor, float], reverse:bool=False) -> Tensor:
+  def _broadcasted(self, fxn:Type[Function], *other:Union[Tensor, float], reverse:bool=False) -> Tensor:
```

Comment:
It's slower to make these lists. Is there a simpler refactor?

Reply:
Un favor me puedes ayudar veo que no puedo recuperar mis cuentas de criptómonedas me pueden apoyar y recuperar todo en si